src/worldInteractions.ts

```
@@ -279,7 +279,9 @@
     const { cursorBlock } = this
 
     let cursorBlockDiggable = cursorBlock
-    if (cursorBlock && (!bot.canDigBlock(cursorBlock) || inAdventure) && bot.game.gameMode !== 'creative') cursorBlockDiggable = null
+    if (!cursorBlock) cursorBlockDiggable = null
+    else if (inAdventure) cursorBlockDiggable = null
```

ESLint

[error] This branch's code block is the same as the block for the branch on line 282.

(sonarjs/no-duplicated-branches)